ql/models/shortrate/onefactormodels/hullwhite.cpp

```diff
@@ -163,7 +164,7 @@ namespace QuantLib {
         Real z = lambda + phi;
 
         Rate futureRate = (100.0-futuresPrice)/100.0;
-        return (1.0-std::exp(-z)) * (futureRate + 1.0/(T-t));
+        return (1.0-std::exp(-z)) * (futureRate + 1.0/deltaT);
```

Comment:
Thanks!  Do you think we should also cover the case T=t?

Reply:
Updated to support T=t as well